### PR TITLE
Add alert rules tutorial

### PR DESCRIPTION
**Pull request by bszwarc**
**Description**

Changes proposed in this pull request:

- Add Alertmanager documentation 
- Add alert rules tutorial

**Related issue(s)**
See also #4886

### CODE REVIEW

File: docs/monitoring/03-01-alertmanager.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Alertmanager
+type: Details
+---
+
+Alertmanager receives and manages alerts coming from Prometheus. It can then forward the notifications on fired alerts to specified channels, such as Slack or VictorOps. 
+
+## Alertmanager configuration
+
+You can configure and manage Alertmanager using the following files:
```
Comment (kazydek) [inline docs/monitoring/03-01-alertmanager.md]: Maybe we could add which configuration file is required in which cases?

File: docs/monitoring/08-03-define-alerting-rules.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    - alert: CPUTempHigh
+      expr: cpu_temperature_celsius > 70 
+      for: 60s
+      labels:
+        severity: critical
+      annotations:
+        description: "CPU temperature exceeds 70 degrees Celsius"
+        summary: "CPU temperature is too high"
+```
+Configure your alert using the following parameters:
```
Comment (kazydek) [inline docs/monitoring/08-03-define-alerting-rules.md]: These are the obligatory things, right?

File: docs/monitoring/03-01-alertmanager.md (hunk 1 above)
Comment (bszwarc) [reply docs/monitoring/03-01-alertmanager.md]: I reworked this one :)

File: docs/monitoring/03-01-alertmanager.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Alertmanager configuration
+
+Use the following files to configure and manage Alertmanager:
+
+* `alertmanager.yaml` which deploys the Alertmananger Pod. 
+* `alertmanager.config.yaml` which you can use to define core Alertmanager configuration and alerting channels. For details on configuration elements, see [this](https://prometheus.io/docs/alerting/configuration/) document.
+* `alertmanager.rules` which lists alerting rules used to monitor Alertmanager's health.
+
+Additionally, Alertmanager instances require a [Secret](../../resources/monitoring/charts/alertmanager/templates/secret.yaml) resource which contains an encoded `alertmanager.yaml.tpl` file. This Secret is picked up during Pod deployment and mounted as `alertmanager.config.yaml`, allowing you to configure alert settings and notifications.
```
Comment (klaudiagrz) [inline docs/monitoring/03-01-alertmanager.md]:
```suggestion
Additionally, Alertmanager instances require a [Secret](../../resources/monitoring/charts/alertmanager/templates/secret.yaml) resource which contains the encoded `alertmanager.yaml.tpl` file. This Secret is picked up during Pod deployment and mounted as `alertmanager.config.yaml`, which allows you to configure alert settings and notifications.
```

File: docs/monitoring/03-01-alertmanager.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    {{ include "alertmanager.yaml.tpl" . | b64enc }}
+{{- range $key, $val := .Values.templateFiles }}
+  {{ $key }}: {{ $val | b64enc | quote }}
+{{- end }}
+```
+To configure the alerts and be able to forward them to different channels, define the parameters: 
+
+| Parameter | Description |
+|-----------|-------------|
+| **name** | Specifies the name of the Secret. The name nas to follow the the `alertmanager-{ALERTMANAGER_NAME}` format. |
```
Comment (klaudiagrz) [inline docs/monitoring/03-01-alertmanager.md]:
```suggestion
| **name** | Specifies the name of the Secret. The name must follow the `alertmanager-{ALERTMANAGER_NAME}` format. |
```

File: docs/monitoring/03-01-alertmanager.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+| **data** | Contains an encoded `alertmanager.yaml.tpl` file which contains all the configuration for alerting notifications provided in the [`alertmanager.config.yaml`](../../resources/monitoring/charts/alertmanager/templates/alertmanager.config.yaml) file.|
```
Comment (klaudiagrz) [inline docs/monitoring/03-01-alertmanager.md]:
```suggestion
| **data** | Contains the encoded `alertmanager.yaml.tpl` file which contains all the configuration for alerting notifications provided in the [`alertmanager.config.yaml`](../../resources/monitoring/charts/alertmanager/templates/alertmanager.config.yaml) file.|
```

File: docs/monitoring/08-03-define-alerting-rules.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+## Prerequisites
+
+This tutorial is a follow-up of the [Observe application metrics](components/monitoring/#tutorials-expose-custom-metrics-in-kyma) tutorial that uses the `monitoring-custom-metrics` example. Follow this tutorial to deploy the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric. That configuration is required to complete this tutorial.
+
+## Steps
+
+Follow these steps to create an alerting rule:
+
+1. Create a PrometheusRule resource holding the configuration of your alerting rule.
```
Comment (klaudiagrz) [inline docs/monitoring/08-03-define-alerting-rules.md]:
```suggestion
1. Create the PrometheusRule resource holding the configuration of your alerting rule. 
```

File: docs/monitoring/08-03-define-alerting-rules.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+This tutorial is a follow-up of the [Observe application metrics](components/monitoring/#tutorials-expose-custom-metrics-in-kyma) tutorial that uses the `monitoring-custom-metrics` example. Follow this tutorial to deploy the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric. That configuration is required to complete this tutorial.
+
+## Steps
+
+Follow these steps to create an alerting rule:
+
+1. Create a PrometheusRule resource holding the configuration of your alerting rule. 
+
+>**NOTE**: Prometheus requires a specific label to identify PrometheusRule definitions. Make sure you set **role** to `alert-rules`.
```
Comment (klaudiagrz) [inline docs/monitoring/08-03-define-alerting-rules.md]:
```suggestion
>**NOTE:** Prometheus requires a specific label to identify PrometheusRule definitions. Make sure you set **role** to `alert-rules`.
```

File: docs/monitoring/08-03-define-alerting-rules.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+        description: "CPU temperature exceeds 70 degrees Celsius"
+        summary: "CPU temperature is too high"
+```
+Configure your alert rule using the following parameters:
+
+| Parameter | Description | Example value |
+|-----------|-------------|---------------|
+| **groups.name** | Specifies the name of the group listing the rules.  | `cpu.temp.rules` |
+| **rules.alert** | Specifies the name of the alert. | `CPUTempHigh`  |
+| **rules.expr** | A PromQL expression specifying the conditions that must be met for the alarm to fire. Specify the expression using Kubernetes [functions](https://prometheus.io/docs/prometheus/latest/querying/functions/) and [metrics](https://github.com/kubernetes/kube-state-metrics/blob/master/Documentation/pod-metrics.md). | `cpu_temperature_celsius > 70`  |
```
Comment (klaudiagrz) [inline docs/monitoring/08-03-define-alerting-rules.md]:
```suggestion
| **rules.expr** | A PromQL expression which specifies the conditions that must be met for the alarm to fire. Specify the expression using Kubernetes [functions](https://prometheus.io/docs/prometheus/latest/querying/functions/) and [metrics](https://github.com/kubernetes/kube-state-metrics/blob/master/Documentation/pod-metrics.md). | `cpu_temperature_celsius > 70`  |
```

File: docs/monitoring/08-03-define-alerting-rules.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+
+
+For more details on defining alerting rules, see [this](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) document.
+
+3. Deploy the alerting rule:
+
+```bash
+kubectl apply -f test-rules.yaml
+```
+4. Run the port-forward command on the monitoring-prometheus service to access the Prometheus dashboard:
```
Comment (klaudiagrz) [inline docs/monitoring/08-03-define-alerting-rules.md]: DO we use any kind of formatting on commands, services, and dashboards names? 🤔

File: docs/monitoring/03-01-alertmanager.md (hunk 3 above)
Comment (bszwarc) [reply docs/monitoring/03-01-alertmanager.md]: Done

File: docs/monitoring/03-01-alertmanager.md (hunk 4 above)
Comment (bszwarc) [reply docs/monitoring/03-01-alertmanager.md]: Done.

File: docs/monitoring/03-01-alertmanager.md (hunk 5 above)
Comment (bszwarc) [reply docs/monitoring/03-01-alertmanager.md]: Done.

File: docs/monitoring/08-03-define-alerting-rules.md (hunk 6 above)
Comment (bszwarc) [reply docs/monitoring/08-03-define-alerting-rules.md]: Done.

File: docs/monitoring/08-03-define-alerting-rules.md (hunk 7 above)
Comment (bszwarc) [reply docs/monitoring/08-03-define-alerting-rules.md]: Done

File: docs/monitoring/08-03-define-alerting-rules.md (hunk 8 above)
Comment (bszwarc) [reply docs/monitoring/08-03-define-alerting-rules.md]: Done.

File: docs/monitoring/08-03-define-alerting-rules.md (hunk 9 above)
Comment (bszwarc) [reply docs/monitoring/08-03-define-alerting-rules.md]: I formatted the command and the service in this case.

**Pull request merged**